benchmark/single-source/NSStringConversion.swift

```diff
@@ -227,4 +241,31 @@ public func run_NSStringConversion_longNonASCII_rebridgeMutable(_ n: Int) {
   innerMutableRebridge(mutableTest, n, 300)
 }
 
+fileprivate func _iterateWithInlineBuffer(string: String, into buffer: UnsafeMutablePointer<CFStringInlineBuffer>) {
+  var result = UInt64(0)
+  let bridged = string as NSString as CFString
+  CFStringInitInlineBuffer(bridged, buffer, CFRangeMake(0, CFStringGetLength(bridged)))
+  for i in 0 ..< CFStringGetLength(bridged) {
+    let char = CFStringGetCharacterFromInlineBuffer(buffer, i)
+    result += UInt64(char)
+  }
+  blackHole(result)
+}
+
+public func run_NSStringConversion_inlineBuffer_UTF8(_ n: Int) {
+  let buffer = UnsafeMutablePointer<CFStringInlineBuffer>.allocate(capacity: 1)
+  for _ in 0 ..< n {
+    _iterateWithInlineBuffer(string: mutableTest, into: buffer)
+  }
+  buffer.deallocate()
+}
+
+public func run_NSStringConversion_inlineBuffer_ASCII(_ n: Int) {
+  let buffer = UnsafeMutablePointer<CFStringInlineBuffer>.allocate(capacity: 1)
+  for _ in 0 ..< n {
+    _iterateWithInlineBuffer(string: mutableTest, into: buffer)
+  }
+  buffer.deallocate()
+}
```

Comment:
This looks the same as `run_NSStringConversion_inlineBuffer_UTF8` -- any reason not to just share the same function in both benchmarks?

Reply:
oh and just change the setup string? good point